packages/eth-json-rpc-provider/src/safe-event-emitter-provider.ts

```diff
@@ -31,18 +31,19 @@ export function convertEip1193RequestToJsonRpcRequest<
 >(
   eip1193Request: Eip1193Request<Params>,
 ): JsonRpcRequest<Params | Record<never, never>> {
```

Comment:
Hmm, this return type is invalid. The `Record<never, never>` does not belong here, no possible return value is represented by that type. Yet if we remove it, we get the type error I referenced in the other thread. How odd.

Maybe we can fix it in a later PR, since it's a pre-existing issue. This type is used later in this file as well.